annotator-core/src/main/java/edu/ucr/cs/riple/core/adapters/NullAwayV2Adapter.java

```diff
@@ -2,30 +2,35 @@
 
 import com.google.common.base.Preconditions;
 import edu.ucr.cs.riple.core.Config;
+import edu.ucr.cs.riple.core.metadata.field.FieldDeclarationStore;
 import edu.ucr.cs.riple.core.metadata.index.Error;
 import edu.ucr.cs.riple.core.metadata.trackers.Region;
 import edu.ucr.cs.riple.injector.location.Location;
 import java.nio.file.Paths;
 import java.util.Arrays;
 
 public class NullAwayV2Adapter extends NullAwayV1Adapter {
-  public NullAwayV2Adapter(Config config) {
-    super(config);
+  public NullAwayV2Adapter(Config config, FieldDeclarationStore fieldDeclarationStore) {
+    super(config, fieldDeclarationStore);
```

Comment:
Shouldn't this pass up `2` as the version somewhere?

Reply:
^

Reply:
4af3e193a813013db2f7674d2b2b170e11606ffa